warehouse/oidc/__init__.py

```diff
@@ -78,3 +78,7 @@ def includeme(config):
 
     # Compute OIDC metrics periodically
     config.add_periodic_task(crontab(minute=0, hour="*"), compute_oidc_metrics)
+
+    # Daily purge expired OIDC-minted API tokens. These tokens are temporary in nature
+    # and expire after 15 minutes of creation.
+    config.add_periodic_task(crontab(minute=0, hour=0), delete_expired_oidc_macaroons)
```

Comment:
Extreme nitpick: there are probably other periodic tasks that run around midnight daily, so we could shift this to `hour=6` or similar to spread the (admittedly very small) load. In practice however, I don't think this will matter much.

Reply:
Changed to `hour=6`